frontend/src/components/Forms/ShoutoutsPage/ShoutoutsPage.module.css

```diff
@@ -6,18 +6,11 @@
   height: calc(100vh - 80px - 25vh);
 }
 
-.shoutoutTitle {
-  text-align: center;
-}
-
-.listsContainer {
-  min-height: 50vh;
-  display: flex;
-}
-
-.listContainer {
-  padding: 2em;
+.shoutoutListContainer {
   width: 50%;
+  align-self: center;
+  margin: auto;
+  height: calc(100vh - 80px - 25vh);
 }
```

Comment:
I think it might be here but the list gets cut off if theres more than 2 shoutouts.
<img width="503" alt="image" src="https://user-images.githubusercontent.com/81320443/197366061-cdf51e2d-dc78-4de5-99f6-b714fbc24e55.png">

Reply:
I was zoomed out so didn't realize. It should be fixed now!